BB-SC-TC/Testing/BB_Digital_Twin.md

```diff
@@ -0,0 +1,80 @@
+
+# Digital Twin
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC-TC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Testing
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
+
+## Known Implementation
+
```

Comment:
Eclipse Ditto

Many many others

Reply:
Agreed. Will be taken care of in the next pull request.